templates/formatter-examples.md.ejs

```diff
@@ -0,0 +1,57 @@
+---
+title: Documentation
+layout: doc
+---
+# ESLint Formatters
+
+ESLint comes with several built-in formatters to control the appearance of the linting results, and supports third-party formatters as well.
+
+The built-in formatter options are:
+
+<% Object.keys(formatterResults).forEach(function(formatterName) { -%>
+* [<%= formatterName %>](#<%= formatterName %>)
+<% }) -%>
+
+## Example Source
+
+Examples of each formatter were created from linting `fullOfProblems.js` using the `.eslintrc` configuration shown below.
+
+#### `fullOfProblems.js`
+
+```js
+function addOne(i) {
+    if (i != NaN) {
+        return i ++
+    } else {
+      return
+    }
+};
+```
+
+#### `.eslintrc`:
+
+```json
+{
+    "extends": "eslint:recommended",
+    "rules": {
+        "consistent-return": 2,
+        "indent"           : [1, 4],
+        "no-else-return"   : 1,
+        "semi"             : [1, "always"],
+        "space-unary-ops"  : 2
+    }
+}
+```
+
+## Output Examples
+<% Object.keys(formatterResults).forEach(function(formatterName) { -%>
+
+### <%= formatterName %>
+<% if (formatterName !== "html") { -%>
+```
+<%- formatterResults[formatterName].result %>
+```
+<% } else {-%>
+[HTML example](html-formatter-example)
```

Comment:
Changing it to an iframe looks cleaner:

![selection_001](https://cloud.githubusercontent.com/assets/4616705/9841871/ddaf4352-5a73-11e5-8b90-7f8e4a5eb0a2.png)